apps/filecoin-site/src/app/_components/Navigation/MobileNavigation.tsx

```diff
@@ -0,0 +1,64 @@
+'use client'
+
+import { useState } from 'react'
+
+import { List, X } from '@phosphor-icons/react'
+import { clsx } from 'clsx'
+
+import { IconButton } from '@filecoin-foundation/ui/IconButton'
+import { LogoLink } from '@filecoin-foundation/ui/LogoLink'
+import { SlideOver } from '@filecoin-foundation/ui/SlideOver'
+
+import { LogoIcon } from '../LogoIcon'
```

Comment:
How about we use the alias here? It reads a bit better.

```suggestion
import { LogoIcon } from '@/components/LogoIcon'
```